src/dynlib/dynlib_linux.c

```diff
@@ -21,11 +22,11 @@ typedef struct _dynlib_context_t
 } dynlib_context_t;
 
 K4A_DECLARE_CONTEXT(dynlib_t, dynlib_context_t);
-static char *generate_file_name(const char *name, uint32_t major_ver, uint32_t minor_ver)
+static char *generate_file_name(const char *name, uint32_t version)
 {
     const char *lib_prefix = "lib";
     const char *lib_suffix = "so";
-    size_t max_buffer_size = strlen(name) + strlen(TOSTRING(DYNLIB_MAX_MAJOR_VERSION)) +
+    size_t max_buffer_size = strlen(name) + strlen(TOSTRING(DYNLIB_MAX_VERSION)) +
                              strlen(TOSTRING(DYNLIB_MAX_MINOR_VERSION)) + strlen(".") + strlen(".") +
```

Comment:
Should this be the string length of "0"?  Or of "DYNLIB_MAX_VERSION"?  I thought DYNLIB_MAX_MINOR_VERSION went away.

Reply:
I made a mistake elsewhere and though Linux was not using minor version in the file name, I will fix this too